addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -658,7 +658,7 @@ export class OdooEditor extends EventTarget {
                             excludedClasses.push(klass);
                         }
                     }
-                    if (excludedClasses.length && excludedClasses.every(c => this.options.renderingClasses.includes(c))) {
+                    if (excludedClasses.every(c => this.options.renderingClasses.includes(c))) {
```

Comment:
`excludedClasses` seems to be the most misleading name for a variable I have ever seen, does it actually mean "classes that were added or removed of the element"? :grimacing: (if yes we can probably change that directly here in stable, it is just a local variable and I am sure I will be debugging this code over the next 3 years so...).

And if this is indeed that, maybe comment the whole section:
```javascript
// First list the classes that were actually removed/added from the element
// ...
// Then ignore the mutation if ...
```
This would actually help me understand this because I don't: the commit you are reverting introduces a test that says "skip the mutations if no changes in state". If I am not confused, "no changes in state" means `excludedClasses.length === 0`... but before their commit we just did not check the `.length` and thus call `.every(...)` directly which means `true`, which means we `continue`... and thus skip the mutation already... no? And with their fix we check the `.length` thus it is falsy since no change of state, so we do not `continue` here and thus consider the mutation? I am very confused: it feels like their fix is doing the opposite of what they meant but a test ensures otherwise :grimacing: